files/grolemund-2016-09.md

```diff
@@ -0,0 +1,65 @@
+    *   understanding small coding details or
+    *   amassing a broad vocabulary of functions.
+
+*   To create a mental model, the teacher must relate new material to previously learned material.
+
+*   Instructors can use multiple choice questions whose answers reveal incorrect mental models to quickly diagnose student backgrounds.
+
+*   Instructors can use concept maps to identify the components of mastering a skill.
+    *   These include the facts that a student must understand and the relationships between those facts.
+    *   With these the instructor can plan out a logical sequence of instruction that lets new facts build on old
+
+*   How to optimize the teaching of code with live coding (taught with examples and anti-examples).
+
+*   How to recognize expert blindspot (i.e., relying on knowledge that the student does not yet have).
+
+*   How to give feedback (practiced with video recording and feedback).
+
+*   How to facilitate participation withan Etherpad or​Google Doc.
+
+## Positive
+
+*   Tries to anchor practice in educational psychology.
+
+*   Introduces the idea of peer-led instruction.
+
+*   Course material is open source, which lets many instructors work together to refine it.
+
+*   Lessons have a useful infrastructure:
+    *   Workshop webpage
+    *   Online course notes and exercises
+￼
+## Negative
+
+*   Includes some material that is not useful, and does not bring material together in a way that can be quickly applied in a classroom.
+
+*   Provides an incomplete picture of what it takes to build a mental model because it does not connect working memory to cognitive load theory or mention a main extension of Cognitive Load Theory (Multimedia Learning Theory):
+    *   Learning tasks involve three types of cognitive load: inherent, generative, and external
+    *   What's missing: a person only has a finite amount of cognitive load to expend at any given moment. Inherent load exceeds this bandwidth, hence you need to scaffold the presentation of complex material. The student must choose to expend generative load, hence you need to show the student how to expend generative load in an efficient way and then motivate them to expend it. Extraneous load diminishes this bandwidth, hence you need to reduce distractions.
+
+*   Multimedia learning theory (not covered)
+    *   Available load is distributed across visual and verbal information channels.
+    *   Material is easier to learn when split simultaneously across different formats.
+    *   It is more difficult to learn when multiple types of information are presented simultaneously in the same format.
+    *   This presents a clear case for visual slides and insights on how to avoid bad slides.
+
+*   Due to the above, SC does not use slides when they would be appropriate, which makes their own material difficult to learn (especially in the instructor workshop).
```

Comment:
Really good points. I should say I don't make traditional slides with many bullet points, just one or a few words and an illustration, which I use to remind me of what I should say.

That being said, I see your points. Curious to hear what Garret has to say about this.